packages/ordeq/tests/snapshots/views/node_inputs_regular_func.snapshot.md

```diff
@@ -0,0 +1,21 @@
+## Resource
+
+```python
```

Comment:
duplicate of packages/ordeq/tests/snapshots/views/view_inputs_regular_func.snapshot.md